driver/Interp.ml

```diff
@@ -387,10 +387,12 @@ let do_external_function id sg ge w args m =
   match camlstring_of_coqstring id, args with
   | "printf", Vptr(b, ofs) :: args' ->
       extract_string m b ofs >>= fun fmt ->
-      print_string (do_printf m fmt args');
+      let formatted = do_printf m fmt args' in
+      let len = (coqint_of_camlint (Int32.of_int (String.length formatted))) in
```

Comment:
The outer parentheses `(coqint_of_camlint ...)` are not needed.  Otherwise, yes, that's a decent (if not super nice) way of getting the length as a 32-bit Coq integer.